ansible/os-devspace.yml

```diff
@@ -0,0 +1,92 @@
+---
+# Playbook for starting a docker instance in the
+# openstack cloud.
+
+## See os-create.yml
+
+## Run:
+# $ source openrc.sh
+# $ ansible-playbook os-devspace.yml -e vm_name=test-devspace -e vm_key_name=yourprofile
+
+- hosts: localhost
+  connection: local
+  #gather_facts: false
+
+  vars:
+
+    vm_image: "CentOS 7 1607"
+    vm_key_name: "jenkins"
+    vm_flavour: m2.large
+    vm_size: 50
+    vm_groups: "ansible-managed,os-image-centos,docker-hosts,devspace"
```

Comment:
What is the conclusion here? Do I need to change anything? Pretty much that is copied from other playbook. Is that ok that will be handled across playbooks?